src/renderer/pages/repository/history.vue

```diff
@@ -87,11 +87,10 @@ export default {
 		this.gitLog();
 	},
 	methods: {
-		gitLog() {
-			gitLog(this.currentRepository).then(result => {
-				this.$store.commit("history/updateLogs", {
-					logs: result
-				});
+		async gitLog() {
+			const logs = await gitLog(this.currentRepository.path);
+			this.$store.commit("history/updateLogs", {
+				logs: logs
```

Comment:
Can you remove the part where you are storing the history inside Vuex Store.

Instead of storing commits in vuex store, we're storing them inside local data property.